client/lib/signup/step-actions/index.js

```diff
@@ -1044,6 +1044,23 @@ export function excludeStepIfProfileComplete( stepName, defaultDependencies, nex
 	}
 }
 
+export async function excludeStepIfWorkspaceListEmpty( stepName, defaultDependencies, nextProps ) {
```

Comment:
Do you think adding the word "P2" in the name would make it a little bit more specific? Like `excludeStepIfP2WorkspaceListEmpty`